utils/netfix/flapping_links.py

```diff
@@ -0,0 +1,300 @@
+#
+# Copyright © 2014-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+"""
+This is the main logic for identifying flapping links
+"""
+
+from datetime import datetime
+import traceback
+import warnings
+
+import pandas as pd
+from netfix_utils import read_and_preprocessing_file, add_partner_info, add_link_hash_id
```

Comment:
to be consistent with other utils when we use it in the plugins
```suggestion
from utils.netfix.netfix_utils import read_and_preprocessing_file, add_partner_info, add_link_hash_id
```